File: C++/Maret 2021/functionReturn.cpp
#include <iostream>
#include <math.h>
using namespace std;

int tambah(int a, int b) {
  return a + b;
}

int kurang(int x, int y) {
  return x - y;
}

int kali(int p, int q) {
  return p * q;
}

int bagi(int c, int d) {
  return c / d;
}

int mod(int e, int f) {
  return e % f;
}

int pangkat(int g, int h) {
  return pow(g, h);
}

int main() {
  printf("Hasil penjumlahan dari 2 + 2 = %i", tambah(2, 2));
  printf("Hasil pengurangan dari 3 - 4 = %i", kurang(3, 4));
  printf("Hasil perkalian dari   6 * 2 = %i", kali(6, 2));
  printf("Hasil pembagian dari   8 / 4 = %i", bagi(8, 4));
  printf("Hasil modulus dari     7 % 3 = %d", mod(7, 3));
  printf("Hasil pangkat dari    5 ** 4 = %i", pangkat(5, 4));
}

File: Pascal/Maret 2021/functionReturn.pas
program functionReturn;
uses crt;

function tambah(a:integer; b:integer):integer;
  begin
    tambah := a + b;
  end;

function kurang(x:integer; y:integer):integer;
  begin
    kurang := x - y;
  end;

function kali(p:integer; q:integer):integer;
  begin
    kali := p * q;
  end;

function bagi(c:integer; d:integer):integer;
  begin
    bagi := c div d;
  end;

function modulus(e:integer; f:integer):integer;
  begin
    modulus := e mod f;
  end;

function pangkat(g:integer; h:integer):integer;
  var 
    i : integer; temp : integer;
  begin
    temp := 1;
    for i := 1 to h do
      begin
        temp := temp * g;
      end;
    pangkat := temp;
  end;

begin
  writeln('Hasil penjumlahan dari 2 + 2 = ', tambah(2, 2));
  writeln('Hasil pengurangan dari 3 - 4 = ', kurang(3, 4));
  writeln('Hasil perkalian dari   6 * 2 = ', kali(6, 2));
  writeln('Hasil pembagian dari   8 / 4 = ', bagi(8, 4));
  writeln('Hasil modulus dari     7 % 3 = ', modulus(7, 3));
  writeln('Hasil pangkat dari    5 ** 4 = ', pangkat(5, 4));
end.
